compiler/control/OMROptions.cpp

```diff
@@ -100,6 +100,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
         TR::Options::setStaticNumeric, (intptrj_t)&OMR::Options::_abstractTimeToReduceInliningAggressiveness, 0, "F%d", NOT_IN_SUBSET },
    {"acceptHugeMethods",     "O\tallow processing of really large methods", SET_OPTION_BIT(TR_ProcessHugeMethods), "F" },
    {"activateCompThreadWhenHighPriReqIsBlocked", "M\tactivate another compilation thread when high priority request is blocked",  SET_OPTION_BIT(TR_ActivateCompThreadWhenHighPriReqIsBlocked), "F", NOT_IN_SUBSET},
+   {"aggressiveLoopVersionerOpts", "O\tUse more loop versioning", SET_OPTION_BIT(TR_aggressiveLoopVersionerOpts), "F" },
```

Comment:
@JamesKingdon this was just _my_ personal preference and as such I did not request changes in my review, but rather a 'review comment' in GitHub terms. To explain my rationale to me `-Xjit:aggressiveLoopVersionerOpts` doesn't really say much. Are we enabling aggressive loop versioner opts? Are we tracing them? Are we disabling them?